src/main/kotlin/no/nav/syfo/behandler/kafka/sykmelding/KafkaBehandlerSykmelding.kt

```diff
@@ -88,14 +88,29 @@ private fun validateReceivedSykmelding(
     return true
 }
 
-private fun createAndStoreBehandlerFromSykmelding(
+private fun createAndStoreBehandlerKontorFromSykmelding(
     receivedSykmeldingDTO: ReceivedSykmeldingDTO,
     behandlerService: BehandlerService,
 ) {
     val partnerId = receivedSykmeldingDTO.partnerreferanse!!
     val behandlerKategori = BehandlerKategori.fromKategoriKode(receivedSykmeldingDTO.legeHelsepersonellkategori)!!
     val arbeidstakerPersonident = Personident(receivedSykmeldingDTO.personNrPasient)
     val sykmeldingBehandler = receivedSykmeldingDTO.sykmelding.behandler
+    val behandlerKontor = BehandlerKontor(
+        partnerId = PartnerId(partnerId.toInt()),
+        herId = receivedSykmeldingDTO.legekontorHerId?.toIntOrNull(),
+        navn = receivedSykmeldingDTO.legekontorOrgName,
+        adresse = sykmeldingBehandler.adresse.gate,
+        postnummer = sykmeldingBehandler.adresse.postnummer?.toString(),
+        poststed = null,
```

Comment:
`adresse.kommune` vil kanskje ikke alltid være det samme som poststed? 🤔
```suggestion
        poststed = sykmeldingBehandler.adresse.kommune,
```

Reply:
Nei, kommune vil nok ikke stemme alltid. Men denne koden er egentlig ikke ny, bare flyttet litt.